Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m

```diff
@@ -410,10 +410,18 @@ - (FBLPromise *)deleteUnsentReports {
   NSOperationQueue *__weak queue = _operationQueue;
   FBLPromise *__weak unsentReportsHandled = _unsentReportsHandled;
   promise = [promise then:^id _Nullable(NSNumber *_Nullable value) {
-    [queue waitUntilAllOperationsAreFinished];
```

Comment:
@samedson @elenadoty Do you remember if blocking UI thread is intentional here or just an accident?

Reply:
I believe this was an oversight. It shouldn't be a problem to delete in the background